src/gravel/controllers/nodes/conn.py

```diff
@@ -61,13 +61,13 @@ async def on_incoming_receive(
         conn: IncomingConnection,
         msg: MessageModel
     ) -> None:
-        logger.debug(f"=> connmgr -- incoming recv: {conn}, {msg}")
```

Comment:
instead of adding the decoration to the log, why just not add a module and function to the debug log format?

Reply:
line number whould good as well

Reply:
we get a lot of pollution in the log. I tried that, and the logs become quite unreadable.

Reply:
Thanks, I see, makes sense. I know debug logs have a lot of garbage, this is why applications in normal mode do not print debug messages.
(This comment was not blocking the PR)